content-repo/extra-docs/packs/Malware_Investigation_&_Response.md

```diff
@@ -0,0 +1,196 @@
+---
+id: malware-investigation-and-response
+title: Malware Investigation and Response
+description: The  Malware Investigation and Response content pack provides a framework for handling malware investigations.
+---
+
+Malware threat is one of the most common cybersecurity challenges facing businesses today. It causes data breaches, hardware failures, and inoperable computers and system networks that can be extremely costly to recover.  
+Malware investigations require security teams to reconcile data from multiple security products like EDRs, sandboxes, malware analysis tools, and threat intelligence providers. 
+Manual investigation wastes valuable time when malware may be propagating within an organization.
+
+The Malware Investigation & Response content pack accelerates the investigation process for endpoint malware incidents and alerts by collecting evidence of malicious behaviors from telemetry data available through EDRs and processing malware analysis reports through sandboxes. Incident layouts also include buttons to remediate activities quickly.    
+The pack closely maps evidence to MITRE ATT&CK to uncover evidence of:
+- Persistence (for example, registry and scheduled jobs)
+- Evasion or tampering (for example and service stop and process kill)
+- Lateral movement (for example, network connections and file share enumeration)
+- PowerShell abuse and command-line analysis
+- Digitally signed files
+
+## What Does This Pack Do?
+* Provides forensic data, including running processes and open network connections at alert detection time.
+* Retrieves files and detonates sandboxes.
+* Analyzes process command-line string for identifying suspicious behavior.
+* Processes select sandbox reports and visualizes the results in layouts.
+* Extracts indicators and provides threat intelligence enrichment.
+* Tags malicious and benign indicators for allow and deny lists for threat prevention and false-positive management.
+* Mirrors incidents between the EDR and Cortex XSOAR and enables incident closure with the EDR.
+* Supports aggregating incidents from EDRs, including multiple alerts and files.
+* Supports fetching malware incidents either from the EDR product or from a SIEM solution.
+
+## Getting Started / How to set up the Pack
+### Use the Deployment Wizard (Available for version 6.8)
+For better user experience and easier onboarding, use the [**Deployment Wizard**](https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-8/cortex-xsoar-admin/marketplace/content-pack-installation/install-a-content-pack/use-the-use-case-deployment-wizard) after installing the content pack on the Marketplace page in Cortex XSOAR (Available for version 6.8).
+  ![Deployment Wizard](../../../docs/doc_imgs/reference/MalwareInvestigationAndResponse/Malware_Investigation_And_Response_layout4.png)
+
+### Manually Configure Your Use Case (From Version 6.5)
```

Comment:
@julieschwartz18 @ssokolovich 
Not sure this is the right terminology
The content is available from 6.5 but you can also manually configure on higher versions

Reply:
Correct, but we want them to use the wizard first (if they are 6.8 and above). That's why we mentioned that the manual config is from 6.5. We can add `From Version 6.5 and above` if you want.